### PR TITLE
Add undefined to optional properties, part W

### PR DESCRIPTION
**Pull request by sandersn**
In preparation for exactOptionalPropertyTypes in Typescript 4.4, add undefined to all optional properties. #no-publishing-comment

This PR covers packages starting with w-.
microsoft/dtslint#335

### CODE REVIEW

File: types/wordpress__block-editor/components/media-placeholder.d.ts [hunk 1]
```diff
@@ -55,30 +55,30 @@ declare namespace MediaPlaceholder {
...
+        onDoubleClick?: MouseEventHandler<HTMLDivElement> | undefined;
         /**
          * Callback called when an upload error happens.
          */
         onError?(message: string): void;
         onSelectURL?(src: string): void;
-        multiple?: T;
-        value?: T extends true ? number[] : number;
+        multiple?: T | undefined;
+        value?: T extends true ? number[] : number | undefined | undefined | undefined;
```
Comment (peterblazejewicz) [inline types/wordpress__block-editor/components/media-placeholder.d.ts:81]:
```suggestion
        value?: T extends true ? number[] : number | undefined;
```
🗡️

**Review by peterblazejewicz — APPROVED** [review]

**Comment by sandersn** [comment]
Looking at the failures:
writable-consumable-stream fails because consumable-stream fails on TS 3.6; this is true on master too, so it's fine to merge with this error.

**Comment by sandersn** [comment]
Also, reviews go much faster when below the 100-file hide-file-by-default limit, so I'll aim for that in future PRs.

**Comment by sandersn** [comment]
Same failure on the second run.

**Pull request merged**